TROUBLESHOOTING.md

```diff
@@ -73,3 +73,39 @@ module.exports = {
 **some-module** and **another-module** will be transformed.
 
 For more information see [here](https://stackoverflow.com/questions/63389757/jest-unit-test-syntaxerror-cannot-use-import-statement-outside-a-module) and [here](https://stackoverflow.com/questions/52035066/how-to-write-jest-transformignorepatterns).
+
+## Tests gets stuck when importing a dependency
```

Comment:
Look like we have duplicated information in this file as well as in `website/docs`. Since you already updated `troubleshooting.md` in `website` folder, we can just clean up this file content and point to https://kulshekhar.github.io/ts-jest/docs/guides/troubleshooting

Reply:
Should I just change the troubleshooting file content to the following ?

```md
Troubleshooting guide have been moved to https://kulshekhar.github.io/ts-jest/docs/guides/troubleshooting
```

Reply:
Yes I think that would be sufficient 👌

Reply:
Done